### PR TITLE
Contact list tweaks

### PR DESCRIPTION
**Pull request by rowanseymour**
* Prevents repeated db hits for date formatting
* Renders missing fields, names and URNs as "--" rather than "None", e.g.

screen shot 2017-03-15 at 14 36 55

becomes...

screen shot 2017-03-15 at 14 37 19

### CODE REVIEW

File: temba/contacts/views.py [hunk 1]
```diff
@@ -123,7 +123,7 @@ def get_queryset(self, **kwargs):
         else:
             qs = group.contacts.all()
 
-        return qs.filter(is_test=False).order_by('-id').prefetch_related('all_groups')
+        return qs.filter(is_test=False).order_by('-id').prefetch_related('org', 'all_groups')
```
Comment (norkans7) [inline temba/contacts/views.py]: I think this is supposed to be select_related instead

Comment (nicpottier) [inline temba/contacts/views.py]: I'll let Rowan comment but it is probably just as fast if not faster as a prefect, it is one more query but less joining.

Comment (rowanseymour) [inline temba/contacts/views.py]: Yeah I'm all about avoiding joins these days! It ends up just one extra hit for an org by its id which is pretty much instant.

Comment (norkans7) [inline temba/contacts/views.py]: yep I see

**Pull request merged**